src/registrar/admin.py

```diff
@@ -959,7 +959,9 @@ class DomainInformationAdmin(ListHeaderAdmin):
                 "classes": ["collapse"],
                 "fields": [
                     "federal_type",
-                    "federal_agency",
```

Comment:
Blocking: My understanding is we keep BOTH fields for now. Make the new field visible only for superadmins.